lib/src/lib/table/components/table-cell-value/table-cell-value.component.ts

```diff
@@ -4,15 +4,21 @@ import { TableCell } from '../../models/table-cell.model';
 @Component({
   selector: 'lab900-table-cell-value',
   template: ` <ng-container *ngIf="cell && cellValue">
-    <span *ngIf="!cell.click" [matTooltipClass]="'lab900-table__mat-tooltip'" [matTooltip]="getMatTooltip()">
+    <span
+      *ngIf="!cell.click"
+      matTooltipClass="lab900-table__mat-tooltip"
+      [matTooltip]="getMatTooltip()"
+      [matTooltipPosition]="this.cell?.cellTooltipOptions ?? 'below'"
```

Comment:
@lab900-frederico-teixeira I believe below is the default position for the tooltip so you shouldn't have to specify it here.

Reply:
how should i do this 🙈 , passing null don't work

Reply:
What about undefined? If that also doesn't work, i guess you'll have to default the cellTooltipOptions somewhere as 'below'. Just don't put it twice here in the template.